iwyu_namespace.h

```diff
@@ -0,0 +1,26 @@
+//===--- iwyu_namespace.h - namespace tracking for includes ---------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef INCLUDE_WHAT_YOU_USE_IWYU_NAMESPACE_H_
+#define INCLUDE_WHAT_YOU_USE_IWYU_NAMESPACE_H_
+
+// Forward declarations
+namespace clang {
+class NamedDecl;
+class NamespaceDecl;
+}  // namespace clang
+
+namespace include_what_you_use {
+void RecordDeclInNamespaces(const clang::NamedDecl *decl);
+
+const clang::NamespaceDecl *GetBestNamespaceDecl(
+    const clang::NamespaceDecl *ns);
```

Comment:
I would remove the parameter name `ns` here, because that would make the whole declaration fit on one 80-col line.